statistics/cmsketch.go

```diff
@@ -298,6 +298,31 @@ func (c *CMSketch) MergeCMSketch(rc *CMSketch) error {
 	return nil
 }
 
+// MergeCMSketch4IncrementalAnalyze merges two CM Sketch for incremental analyze. Since there is no value
+// that appears partially in `c` and `rc` for incremental analyze, it uses `max` to merge them.
+// Here is a simple proof: when we query from the CM sketch, we use the `min` to get the answer:
+//   (1): For values that only appears in `c, using `max` to merge them affects the `min` query result less than using `sum`;
+//   (2): For values that only appears in `rc`, it is the same as condition (1);
+//   (3): For values that appears both in `c` and `rc`, if they do not appear partially in `c` and `rc`, for example,
+//        if `v` appears 5 times in the table, it can appears 3 times in `c` and 5 times in `rc`, then `max` also gives the correct answer.
```

Comment:
Since `rc` is the old CMSketch and `c` is the new one and this column is `append only` and `increase only`, appear times of `v` in `c` should be always equal or large than `rc`, so it should be ```it can appears 5 times in `c` and 3 times in `rc` ```?